examples/avian2d_colliders.rs

```diff
@@ -105,26 +114,34 @@ fn boulders_spawn(
     game_assets: Res<GameAsset>,
     image_assets: Res<Assets<Image>>,
 ) {
-    let sprite_handle = game_assets.image_handles.get("boulders");
-    if sprite_handle.is_none() {
+    let Some(sprite_handle) = game_assets.image_handles.get("boulders") else {
         return;
-    }
-    let sprite_image = image_assets.get(sprite_handle.unwrap()).unwrap();
-
-    let edges = Edges::from(sprite_image);
-    let coord_group = edges.multi_image_edge_translated();
-    let colliders = generate_colliders(sprite_image, ColliderType::ConvexPolyline, true);
+    };
+    let sprite_image = image_assets.get(sprite_handle).unwrap();
+    let edges = Edges::try_from(sprite_image).unwrap();
+    let polygons = edges.iter();
+    let colliders = AbstractCollidersBuilder::try_from(sprite_image)
+        .unwrap()
+        .absolute()
+        .polyline()
```

Comment:
It's unclear to me why, but the `polyline` collider seems to be the problem. Gravity works if this is changed to `.convex_polyline()`